### PR TITLE
Update atomist dependencies

### PR DESCRIPTION
**Pull request by lievendoclo**
[atomist:generated] [atomist:autofix=typescript_header]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +68,7 @@
     "fmt:gql": "prettier --write \"**/*.graphql\"",
     "git:info": "atomist git",
```
Comment (ddgenome) [inline package.json]: This is `atm-git-info` or something like that now.

File: src/atomist.config.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@
  */
 
 import { Configuration } from "@atomist/automation-client";
-import { ingester } from "@atomist/automation-client/graph/graphQL";
-import * as secured from "@atomist/automation-client/secured";
+import { ingester } from "@atomist/automation-client/lib/graph/graphQL";
+import * as secured from "@atomist/automation-client/lib/secured";
```
Comment (ddgenome) [inline src/atomist.config.ts]: Can these not be imported from the index?

**Review by ddgenome — COMMENTED** [review]
Looks good. A couple minor things.

File: package.json [hunk 3]
```diff
@@ -18,12 +18,12 @@
     "sentry"
   ],
   "dependencies": {
-    "@atomist/cli": "0.2.2-20180731183747",
-    "@atomist/automation-client": "0.19.0-20180731181755",
-    "@atomist/automation-client-ext-dashboard": "0.1.0-20180705210657",
-    "@atomist/automation-client-ext-logzio": "0.1.0-20180628224529",
-    "@atomist/automation-client-ext-raven": "0.1.0-20180613131132",
-    "@atomist/slack-messages": "^0.12.1"
+    "@atomist/cli": "^1.2.0",
```
Comment (cdupuis) [inline package.json]: Why is the cli here?

File: src/atomist.config.ts (hunk 2 above)
Comment (lievendoclo) [reply src/atomist.config.ts]: Handled. Thx.

File: package.json (hunk 1 above)
Comment (lievendoclo) [reply package.json]: Replaced.

File: package.json [hunk 4]
```diff
@@ -35,16 +35,17 @@
...
-    "ts-node": "^7.0.0",
-    "tslint": "^5.10.0",
-    "typedoc": "^0.10.0",
-    "typescript": "2.9.2",
-    "typescript-formatter": "^7.2.0"
+    "ts-node": "^8.0.2",
+    "tslint": "^5.12.1",
+    "typedoc": "^0.14.2",
+    "typescript": "^3.3.3",
+    "typescript-formatter": "^7.2.2"
```
Comment (cdupuis) [inline package.json]: formatter can go.

File: package.json (hunk 3 above)
Comment (lievendoclo) [reply package.json]: I just updated the dependencies. I guess it can go, but I didn't want to start removing.

**Review by cdupuis — APPROVED** [review]

**Review by ddgenome — APPROVED** [review]
It would be ideal to import everything possible from index, but let's get this merged.

File: package.json (hunk 3 above)
Comment (ddgenome) [reply package.json]: It used to be needed when the build helpers were in the CLI and not provided by automation-client.

**Pull request merged**